grub-core/net/bootp.c

```diff
@@ -31,6 +31,8 @@
 #define GRUB_NET_BOOTP_ARCH 0x0007
 #elif defined(GRUB_MACHINE_EFI) && defined(__aarch64__)
 #define GRUB_NET_BOOTP_ARCH 0x000B
+#elif defined(__powerpc__)
```

Comment:
Thanks @marineam ' explain. I glanced though the grub upstream code and according to your comments, I suggest that we revert [fbf6567](https://github.com/coreos/grub/commit/fbf65674a471d12e43f4050086969f417e00891f) as upstream did. Lastly, Happy Christmas Day.